test/core/transport/chttp2/hpack_encoder_test.cc

```diff
@@ -219,6 +237,117 @@ TEST(HpackEncoderTest, TestBasicHeaders) {
   delete g_compressor;
 }
 
+MATCHER(HasLiteralHeaderFieldNewNameFlagIncrementalIndexing, "") {
+  constexpr size_t kHttp2FrameHeaderSize = 9u;
+  constexpr uint8_t kLiteralHeaderFieldNewNameFlagIncrementalIndexing = 0x40;
```

Comment:
Can you add a brief comment about why it needs to be 0x40

Reply:
I feel like the long variable name should be somehow self-explanatory. But it should be ok to add comments here.